aioshelly/const.py

```diff
@@ -918,6 +922,34 @@ class ShellyDevice:
         gen=GEN3,
         supported=True,
     ),
+    MODEL_1PM_G4: ShellyDevice(
+        model="S4SW-001P16EU",
```

Comment:
Why we always write the model manually instead of using MODEL_1PM_G4 as it's already there ?

Reply:
Good question, I don't know 😄